src/content/administration/dashboard.mdx

```diff
@@ -95,3 +145,29 @@ ${MY_PASSWORD}
 ```
 
 where `MY_PASSWORD` can be a local environment variable or an Okteto Secret, and it gets subsituted by Okteto before mounting the file as a build secret.
+
+## Catalog
+
+This section of the Admin dashboard enables you to create a [catalog](/administration/catalog/) of pre-defined development environments for anyone in your Okteto instance to deploy quickly from their `Launch Dev Environment` screen. You can reduce the amount of time it takes a developer to get up and running by defining these catalog entries so they're available for developers to select and launch with minimal to no configuration.
+
+<p align="center"><Image src="/docs/administration/images/catalog_table_view.png" alt="Catalog table view" width="900" /></p>
+
+:::info
+
+This feature is currently available only for Self-Hosted customers.
```

Comment:
It's available everywhere the admin view is available. That means it's only disabled in Okteto Cloud

Reply:
Good catch, thank you! Will update